lib/node_modules/@stdlib/blas/base/sdot/docs/repl.txt

```diff
@@ -2,8 +2,8 @@
 {{alias}}( N, x, strideX, y, strideY )
     Computes the dot product of two single-precision floating-point vectors.
 
-    The `N`, `strideX`, and `strideY` parameters determine which elements in `x`
-    and `y` are accessed at runtime.
+    The `N` and stride parameters determine which elements in the
+    strided arrays are accessed at runtime.
```

Comment:
```suggestion
    The `N` and stride parameters determine which elements in the strided
    arrays are accessed at runtime.
```

Need to ensure that REPL text file descriptions wrap at 80 chars. When editing descriptions, this may require manually rewrapping lines.